labs/contentsquare-integration-web/README.md

```diff
@@ -17,12 +17,15 @@ This integration is using [Optimizely's Custom Analytics feature](https://help.o
  6. Paste the following code: 
 
 ```json
-{"plugin_type": "analytics_integration",
-"name": "ContentSquare Integration",
-"form_schema": [],
-"description": "This integration allows you to send the experiment and variation data from Optimizely Web to ContentSquare",
-"options": {
-"track_layer_decision": "function sendTestToCS(csKey, csValue) {\n window._uxa = window._uxa || [];\n window._uxa.push([\"trackDynamicVariable\", {key: csKey, value: csValue} ]);\n}\n\nfunction callback(context) {\n\tif (!disableCallback) {\n\t\tdisableCallback = true;\n\n\t\tif (window.CS_CONF) {\n CS_CONF.integrations = CS_CONF.integrations || [];\n CS_CONF.integrations.push(\"Optimizely Web\");\n\t\t}\n\t}\n}\n  \nvar disableCallback = false;\nwindow._uxa = window._uxa || [];\n_uxa.push([\"afterPageView\", callback]);\n\nvar csPrefix = 'AB_Opti_';\nvar csKey = csPrefix + campaignId + '_' + experimentId;\nvar csValue = variationId;\n\nsendTestToCS(decodeURI(csKey), decodeURI(csValue));"}}
+{
+  "plugin_type": "analytics_integration",
+  "name": "ContentSquare",
+  "form_schema": [],
+  "description": "",
```

Comment:
Add back the description

`This integration allows you to send the experiment and variation data from Optimizely Web to ContentSquare`

Reply:
@asaschachar I have added, thanks for the feedback!